c/src/ml-api-inference-single.c

```diff
@@ -1385,7 +1389,14 @@ _ml_single_invoke_internal (ml_single_h single,
     single_h->output = *output;
   }
 
-  single_h->input = input;
+  /**
+   * Clone input data here to prevent use-after-free case.
+   * We should release single_h->input after calling __invoke() function.
+   */
+  status = ml_tensors_data_clone (input, &single_h->input);
+  if (status != ML_ERROR_NONE)
+    goto exit;
```

Comment:
Is this memcpy really required?

The invoke APIs are synchronous.

Reply:
The exception case when releasing input data handle while invoking a model. Now tf-lite subplugin directly uses the input data in single-shot.